app/modules/feeds-and-posts.js

```diff
@@ -0,0 +1,57 @@
+import { Promise } from 'rsvp';
+
+let fpPromise = null;
+
+/**
+ * Gets the Feeds & Posts ES Module, by first appending the F&P library to the page
+ *
+ * @returns {Promise} Promise of module
+ */
+function getModule() {
```

Comment:
in ember cli you can use ember auto import to import modules from npm (also dynamically)
https://github.com/ef4/ember-auto-import#dynamic-import

can we install feeds-and-posts lib in mobile-wiki and use it instead of loading it from separate service?
also in lib.min.js there is 706KB of code - I suspect not all of it is needed, maybe using auto import would help with this as well?

Reply:
does this line mean we'll load Design System twice in mobile-wiki https://github.com/Wikia/fepo/blob/release-8/app/webpack/loaders.js#L38 ?

Reply:
ouch forgot that we still load AdEngine2 that stops us from using ember-auto-import - so the custom logic to load the lib has to stay for now

Reply:
@hakubo DS import should be resolved today - the lib.min.js shouldn't load DS at all.